src/components/job/JobApproveReject.jsx

```diff
@@ -67,10 +67,14 @@ export default function JobApproveReject({ jobId }) {
             ></textarea>
           </div>
           <div className="w-full flex flex-row justify-center mt-5">
-            <label htmlFor="feedback-display" className="btn btn-primary mr-5">
+            <label htmlFor="feedback-display" className="btn btn-secondary mr-5">
```

Comment:
just curious: what' the good for using label here? usually we are using `button`